package.json

```diff
@@ -73,5 +73,8 @@
     "webpack-cli": "^3.1.2",
     "webpack-dev-server": "^3.1.10",
     "webpack-merge": "^4.1.4"
+  },
+  "dependencies": {
+    "@warren-bank/electron-win-portable-paths": "^0.1.0"
```

Comment:
dependencies should be added to `src/package.json` because we use two package.json structure. And `src/yarn.lock` also should be committed to restrict package version.